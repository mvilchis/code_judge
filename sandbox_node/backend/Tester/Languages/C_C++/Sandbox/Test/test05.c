/* Test that printf works
 * You'd be amazed how many systems calls are used in modern
 * libcs just to print text to stdout. */

#include <stdio.h>

int main(void) {
	int x = 5;
	float y = 6.1;
	char s[] ="entro";
	printf("Hello, world %d %.1f %s\n", x, y,s);
	return 0;
}
