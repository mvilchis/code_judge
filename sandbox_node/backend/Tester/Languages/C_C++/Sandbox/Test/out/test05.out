Hello,world56.1entro